lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -109,6 +109,21 @@ class TraceElements extends Gatherer {
     return topFive;
   }
 
+  /**
+   * @param {LH.Gatherer.PassContext['driver']} driver
+   * @param {number} backendNodeId
+   * @return {Promise<string|undefined>}
+   */
+  async resolveNodeId(driver, backendNodeId) {
```

Comment:
can this move to the Driver class?

Reply:
it certainly can although nothing else will use it in this way, is your concern mostly forward looking in making the same mistake?

Reply:
either way, I think this was worth the cleanliness, so done :)